### PR TITLE
README: describe how we handle feature requests

### PR DESCRIPTION
**Pull request by berendt**
Signed-off-by: Christian Berendt <berendt@osism.tech>

### CODE REVIEW

**Comment by berendt** [comment]
@fkr @garloff @ra-beer

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
+As an active project within the Sovereign Cloud Stack and since we are used there as a reference
+implementation for the infrastructure layer, we coordinate very closely there. It is very likely
+that we will first need to evaluate and align feature related issues in the context of the Sovereign
+Cloud Stack. As part of the Sovereign Cloud Stack project, various assumptions were made for the
+reference implementation which cannot be changed or extended independently as a project. If a feature
+related Issue requires a review in the Sovereign Cloud Stack project, this will be noted in the issue
+and the issue will be tagged with an SCS tag. Afterwards, a review of the issue will take place in
+one of the upcoming project meetings. Appointments are currently taking place weekly on Wednesdays at
+10 a.m. (see https://scs.community/contribute/).
```
Comment (fkr) [inline README.md:13]: 10 a.m. CET

**Review by fkr — APPROVED** [review]
added couple minor suggestions.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
 # issues
 
 This repository is used for issues that are cross-repository or not bound to a specific repository.
+
+As an active project within the Sovereign Cloud Stack and and as the reference implementation for
+the infrastructure layer, we coordinate very closely there. It is very likely
+that we will first need to evaluate and align feature related issues in the context of the Sovereign
+Cloud Stack. As part of the Sovereign Cloud Stack project, various assumptions were made for the
+reference implementation which cannot be changed or extended independently as a project. If a feature
```
Comment (garloff) [inline README.md:9]: ... which OSISM does not want to or sometimes can not change indepdently from SCS.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
 # issues
 
 This repository is used for issues that are cross-repository or not bound to a specific repository.
+
+As an active project within the Sovereign Cloud Stack and and as the reference implementation for
```
Comment (ra-beer) [inline README.md:5]: ... Sovereign Cloud Stack and ~~and~~ as the reference...

**Review by ra-beer — APPROVED** [review]
Looks really good to me!

**Comment by berendt** [comment]
@fkr @garloff Good to merge?

**Review by fkr — APPROVED** [review]
LGTM!

**Comment by fkr** [comment]
For me it looks good!

**Pull request merged**